ibis/pandas/execution/window.py

```diff
@@ -66,6 +67,67 @@ def _post_process_group_by_order_by(series, parent, order_by, group_by):
     return series
 
 
+@functools.singledispatch
+def get_aggcontext(
+    window, *, operand, operand_dtype, parent, group_by, order_by
+):
+    raise NotImplementedError(
+        f"get_aggcontext is not implemented for {type(window)}"
```

Comment:
Just a suggestion, but may be just the name of the class (e.g. `int` instead of `<class 'int'>`)?
```suggestion
        f"get_aggcontext is not implemented for {type(window).__name__}"
```